### PR TITLE
Prepare for release

### PR DESCRIPTION
**Pull request by g4s8**
#49 - few minor improvements for release:
 - added `AstoDocker` implementation
 - moved repo name from `Repo` method paramter to implementation constructor
 - removed unused interfaces
 - cleanup javadocs

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#50](https://github.com/artipie/docker-adapter/issues/50) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/1c5c1711-912d-4cfb-9b63-16719370b413); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/main/java/com/artipie/docker/asto/AstoDocker.java [hunk 1]
```diff
@@ -22,19 +22,40 @@
...
+import com.artipie.docker.Repo;
+import com.artipie.docker.RepoName;
 
 /**
- * Registry reference path URI.
+ * Asto {@link Docker} implementation.
  * @since 0.1
  */
-public interface RefPath {
+public final class AstoDocker implements Docker {
```
Comment (carlosmiranda) [inline src/main/java/com/artipie/docker/asto/AstoDocker.java]: @g4s8 perhaps we should create a unit test for this? At least a puzzle.

Comment (g4s8) [reply src/main/java/com/artipie/docker/asto/AstoDocker.java]: @carlosmiranda ok, but how would you suggest testing this object? It's just creates `Asto*` implementations for `BloblStore` and `Repo` interfaces. Is it a good practice to test the types of returning objects using `instanceof`?

Comment (carlosmiranda) [reply src/main/java/com/artipie/docker/asto/AstoDocker.java]: @g4s8 I suppose we can do that to start with. There's `Matchers.instanceOf`. At least with the test in place we can expand it with other checks if it ever becomes necessary.

**Comment by g4s8** [comment]
@carlosmiranda fixed, please see: https://github.com/artipie/docker-adapter/pull/50/commits/45e39760e19fcdbda35ccdb371481e80c143f11d

**Comment by 0crat** [comment]
Job was finished in 22 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Job audit: performer @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)[/z](https://www.0crat.com/u/carlosmiranda) didn't make CR comments

**Comment by 0crat** [comment]
Order was finished: +20 point(s) just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Pull request merged**